### DISCUSSION TITLE
Add Email Input for Connection Creation with Notification Feature

### DISCUSSION THREAD

**Discussion by luongnv275** [apache/airflow#36101] [Ideas]
Currently, when creating a connection, there is no option to input an email for receiving notifications upon data transmission. Proposing the addition of an email input and notification functionality upon data transmission completion.

**Comment by potiuk** [comment]
It's' not question of field, That would likely require implementation of notifications for some 1500 or so hooks with likely few thousands methods.

Currently Airflow "framwork" is semantically unaware about "hook" completing data tramsmission so you'd effectively have to implement such notification in all the hooks individually and it will not cover all the custom hooks you might want to have.

But if you or your company would like to donate their time and contribute changes  to Airflow implementing it (likely at least few hundreds PR to review and merge) - then you can potentially attempt to add this new feature. It would likely require reaching to devlist and getting AIP proposing it as it would fundamentally change the Hook interface. 

See "community" tab for all relevant information on devlist, aip etc.
